src/services/includers/batteries/openapi/generators/traverse.ts

```
@@ -234,3 +234,18 @@ function merge(value: OpenJSONSchemaDefinition): OpenJSONSchema {
 function isRequired(key: string, value: JSONSchema6): boolean {
     return value.required?.includes(key) ?? false;
 }
+
+function inferType(value: OpenJSONSchema): JSONSchema6['type'] {
```

as far as i understand `inferType` might return `undefined` yet i can't find you handling that scenario

---

This PR is about adding a type calculation for enum, this function does not handle undefined.

---

> This PR is about adding a type calculation for enum

1. you are adding function to infer type from value.

2. this function has undefined as one of its return values

3. either handle function returning undefined when you call it or throw error instead of returning undefined inside of the function.

---

fixed